benchmarks/renderToString.js

```diff
@@ -4,31 +4,59 @@
  * compare renderToString
  */
 
-const cp = require('child_process');
+const Benchmark = require('benchmark');
 
-const limit = 10;
+const Rax = require('rax');
+const raxRenderToString = require('rax-server-renderer').renderToString;
+const React = require('react');
+const ReactDOMServer = require('react-dom/server');
```

Comment:
Yes, requiring react-dom/server from the dist directory was never really supported, and it no longer works. 

As I said in my comment, if you merge this PR as is, I will happily make a PR that does the optimizations on top of it.